### DISCUSSION TITLE
New to Fossify File Manager, I have some questions

### DISCUSSION THREAD

**Discussion by Haruto277-Dantes** [FossifyOrg/General-Discussion#484] [Q&A]
Hello! I've just started using Fossify File Manager, is there anyway to "hide" the important file folders? (for example the Android folder?) for example within the Samsung File Manager app, there is an "Essentials" tab which restricts the user to only be able to see just the DCIM, Download, Music and Pictures folder. Can I also do this on Fossify? when I tried to "hide" the Android folder it pulls up some sort of error.

**Comment by Haruto277-Dantes** [comment]
Thank you to the Fossify team for making a good app!!!!

**Comment by Aga-C** [comment]
Hiding in Fossify File Manager is a system-wide hide, so it's impossible to set it for system folders.

**Reply by Haruto277-Dantes** [reply]
Thank you!!!! I'll continue to use the Samsung Files for the time being then, Thank you so much!!!!
